### PR TITLE
use real location hierarchy in ICDS dashboard tests

### PR DESCRIPTION
**Pull request by czue**
##### SUMMARY

This switches tests to use the same location hierarchy as the production environment. Which should hopefully make things a bit more consistent in our test setup and pave the way for future cleanup.

### CODE REVIEW

File: custom/icds_reports/tests/test_export_data.py [hunk 1]
```diff
@@ -2080,6 +2081,22 @@ def test_beneficiary_export(self):
...
+                            'd1'
+                        ],
+                        [
+                            'Block',
+                            'b1'
+                        ],
+                        [
+                            'Supevisor',
+                            's1'
+                        ],
```
Comment (czue) [inline custom/icds_reports/tests/test_export_data.py]: this was the only non-obvious change. now that the AWC has a proper parent line these all get included here. I assume that's the correct behavior, and the previous test case was kind of wonky?

**Comment by czue** [comment]
@calellowitz ping

**Review by Rohit25negi — APPROVED** [review]
Thanks for this Cleanup Cory. Looks good to me. 
Only one precautious thing. I remember changing in location hierarchy may lead citus to fail. I am not sure how to validate that. @calellowitz  can you comment on it.

**Comment by czue** [comment]
>  I remember changing in location hierarchy may lead citus to fail. I am not sure how to validate that.

This might explain a test failure I was seeing locally that didn't show up here (I just assumed travis was on citus until today). Will wait for @calellowitz to weigh in after he looks at switching tests over. I'll also let you know in 20ish minutes what the exact failure was and also whether it also shows up without this change.

**Comment by czue** [comment]
This is the error I get locally with the change:
```
======================================================================
ERROR: custom.icds_reports.tests.test_aggregation_script:LocationAggregationTest.test_agg
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/czue/src/dimagi/commcare-hq/custom/icds_reports/tests/test_aggregation_script.py", line 586, in test_agg
    self.helper.aggregate(cursor)
  File "/home/czue/src/dimagi/commcare-hq/custom/icds_reports/utils/aggregation_helpers/monolith/awc_location.py", line 106, in aggregate
    cursor.execute(self.create_local_table())
  File "/home/czue/.virtualenvs/commcare-hq-py3/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/czue/.virtualenvs/commcare-hq-py3/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/czue/.virtualenvs/commcare-hq-py3/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/czue/.virtualenvs/commcare-hq-py3/lib/python3.6/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
django.db.utils.InternalError: cannot INSERT rows from a distributed query into a local table
HINT:  Consider using CREATE TEMPORARY TABLE tmp AS SELECT ... and inserting from the temporary table.
```

**Comment by czue** [comment]
(same error on master so I think this is safe to merge)

**Pull request merged**